TombEngine/Game/effects/lightning.cpp

```diff
@@ -2,7 +2,7 @@
 #include "Game/effects/lightning.h"
 
 #include "Game/animation.h"
-#include "Game/effects/bubble.h"
+#include "Game/effects/Bubble.h"
```

Comment:
I see Bubble class was already included in the lightning.cpp effect. But does it really need it?. I can't find where it's using it.

Maybe wherever it was being used, was replaced by particles effects or something else?.

Reply:
lightning.cpp has several pointless includes. Will remove this one!